mypy_django_plugin/transformers/auth.py

```diff
@@ -0,0 +1,40 @@
+from mypy.plugin import FunctionSigContext
+from mypy.types import CallableType, FunctionLike, Instance, UnionType
+
+from mypy_django_plugin.django.context import DjangoContext
+from mypy_django_plugin.lib import helpers
+
+
+def transform_user_passes_test(ctx: FunctionSigContext, django_context: DjangoContext) -> FunctionLike:
```

Comment:
Can't we name this function a bit more specifically?

e.g.

```suggestion
def attach_AUTH_USER_MODEL_to_user_passes_test(ctx: FunctionSigContext, django_context: DjangoContext) -> FunctionLike:
```